src/screens/VaultScreen.js

```diff
@@ -13,11 +14,305 @@ export default class VaultScreen extends React.Component {
     ),
   };
 
+  async componentDidMount() {
+    await Font.loadAsync({
+      'source-sans-pro-black': require('../.././assets/fonts/SourceSansPro-Black.ttf'),
+      'source-sans-pro-black-italic': require('../.././assets/fonts/SourceSansPro-BlackItalic.ttf'),
+      'source-sans-pro-bold': require('../.././assets/fonts/SourceSansPro-Bold.ttf'),
+      'source-sans-pro-bold-italic': require('../.././assets/fonts/SourceSansPro-BoldItalic.ttf'),
+      'source-sans-pro-extra-light': require('../.././assets/fonts/SourceSansPro-ExtraLight.ttf'),
+      'source-sans-pro-extra-light-italic': require('../.././assets/fonts/SourceSansPro-ExtraLightItalic.ttf'),
+      'source-sans-pro-italic': require('../.././assets/fonts/SourceSansPro-Italic.ttf'),
+      'source-sans-pro-light': require('../.././assets/fonts/SourceSansPro-Light.ttf'),
+      'source-sans-pro-light-italic': require('../.././assets/fonts/SourceSansPro-LightItalic.ttf'),
+      'source-sans-pro-regular': require('../.././assets/fonts/SourceSansPro-Regular.ttf'),
+      'source-sans-pro-semibold': require('../.././assets/fonts/SourceSansPro-SemiBold.ttf'),
+      'source-sans-pro-semibold-italic': require('../.././assets/fonts/SourceSansPro-SemiBoldItalic.ttf'),
```

Comment:
Can we use google fonts instead of loading the font files ourselves? https://fonts.google.com/specimen/Source+Sans+Pro

Reply:
JK this is proper :P We should probably move all the font stuff into a separate style file though.

Reply:
@clairelin135 ^ let's move the fonts into a separate file